package/base-files/files/etc/init.d/done

```diff
@@ -11,6 +11,24 @@ boot() {
 		sh /etc/rc.local
 	}
 
+	[ -d /sys/firmware/efi ] && {
+		mkdir /tmp/boot
+		mount PARTLABEL=openwrt_boot -o rw,noatime /tmp/boot
+		grub-editenv /tmp/boot/boot/grub/grubenv set attempt=0
+		if grep -q openwrt_rootfs_alt /proc/cmdline; then
+			if grep -q '^set default="0"' /tmp/boot/boot/grub/grub.cfg; then
+				sed -e 's:^set default="0":set default="1":' \
+				    -e 's:^set fallback="1":set fallback="0":' \
+				    -i /tmp/boot/boot/grub/grub.cfg
+			fi
+		elif grep -q '^set default="1"' /tmp/boot/grub/grub.cfg; then
+			sed -e 's:^set default="1":set default="0":' \
+			    -e 's:^set fallback="0":set fallback="1":' \
+			    -i /tmp/boot/grub/grub.cfg
+		fi
```

Comment:
I'm disliking this `set default="0"`/`set default="1"` portion.  From looking at another portion I suspect there is a better strategy.